pyanaconda/ui/gui/spokes/software.py

```diff
@@ -156,7 +160,12 @@ def _payload_error(self):
         hubQ.send_message(self.__class__.__name__, payloadMgr.error)
 
     def _apply(self):
-        if self.environment and not (flags.automatedInstall and self.data.packages.seen):
+        # Environment have to be set by GUI but not by kickstart
```

Comment:
Rewording suggestion:

`Environment needs to be set during a GUI installation, but is not required for a kickstart install (even partial)`